vercel.json

```
@@ -21,6 +21,10 @@
       "source": "/chat",
       "destination": "https://discord.gg/sNgJkMg"
     },
+    {
+      "source": "/roadmap",
+      "destination": "https://www.notion.so/verto/48f99dfb0ba744f5b5257185f53ad7c4?v=59433d7065d44fad9c4191aba6635c5f"
```

```suggestion
      "destination": "https://www.notion.so/verto/48f99dfb0ba744f5b5257185f53ad7c4"
```

need to remove the `?v=...` query string otherwise it won't get updated and will be locked to one version.

---

Thank you for catching this!